docs/collection_key_definition.md

```diff
@@ -16,6 +16,7 @@ The schema for each collecion we load into our database is defined in schema-con
 | gaf                                               | edge | go_gene_product_links               | Y | {annotation_dict}  |
 | gtex splice variant to gene association           | edge | variant_gene_links                  | N | {variant_id}\_{gene_id}\_{ontology_term_id} |
 | gtex variant to gene expression association       | edge | variant_gene_links                  | N | {variant_id}\_{gene_id}\_{ontology_term_id} |
+| variant to phenotypic feature association       | edge | variants_phenotypes                  | Y | {variant_id}\_{ontology_term_id} |
```

Comment:
collection key definition doesn't match since three collections are added. Need to add key definition for studies, studies_variants, and studies_variants_phenotypes.